### DISCUSSION TITLE
Welcome to lvgl_micropython Discussions!

### DISCUSSION THREAD

**Discussion by kdschlosser** [lvgl-micropython/lvgl_micropython#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mark-hoy** [comment]
I hadn't looked at the README in a while since I usually just git clone and compile on my OSX system and then debugged my code as lvgl_micropython has been building and working well for me.  The readme and pulled code is looking VERY nice and everything I have tried just worked.  I'd like to thank you @kdschlosser for all this hard work and help.

Recently I got a cheap mini PC with Windows 11, yesterday I updated it and installed WSL2. I then installed Ubuntu 24 and proceeded to install a build chain.  

The thing I missed was the Espressif IDE for ESP32 builds. 
```
$ git clone --recursive https://github.com/espressif/esp-idf.git
$ cd esp-idf
$ ./install.sh
```

I also compiled for unix and ran the resulting ./lvgl_micropython_unix, loaded up the suggested sample code and voila, a slider. 

I am supporting a graphics install on a 7” Esp32 Crowpanel that connects into FluidNC

**Reply by kdschlosser** [reply]
You don't have to do anything with the ESP-IDF. It is all done for you by the build script.

**Reply by kdschlosser** [reply]
And TY for the nice comments. It is nice to know that people do appreciate how it just works using a single command to compile. That is the single most important thing about running any flavor of Python, it needs to be easy to use.

**Reply by mark-hoy** [reply]
> You don't have to do anything with the ESP-IDF. It is all done for you by the build script.

Ah, I see my problem, to build ESP32 I needed to "sudo apt install python3-venv".

**Comment by otosan-maker** [comment]
hello 

I Have asked some questions in the past in the lvgl forum, but I think it is the right place. Two years ago I made a calculator project, it runs an old micropython version with an esp32. I have developed a new hardware platform, with an esp32s3, and I want to go to esp-idf 5 and the latest mycropython. 
I have several issues, I will ask in the forum, please bear with me.
